### PR TITLE
https://w3.rchland.ibm.com/projects/bestquest/?defect=SW554306

### PR DESCRIPTION
**Pull request by adangster**
change 
<id>VPD_TYPE</id>
		<default>RSPD</default>

<id>WRITE_PAGE_SIZE</id>
		<default>0x08</default>

---- other changes is due to SW2 auto updates to latest common src file.

### CODE REVIEW

**Review by mderkse1 — APPROVED** [review]

File: Rainier-4U-MRW.xml [hunk 1]
```diff
@@ -50902,6 +50918,10 @@
 		<id>SYSTEM_VDM_DISABLE</id>
 		<default>0x00</default>
 	</attribute>
+	<attribute>
+		<id>SYS_CLOCK_INTEGRATED_SPARES</id>
```
Comment (dcrowell77) [inline Rainier-4U-MRW.xml:50922]: Looks like you accidentally reverted the value for SYS_CLOCK_INTEGRATED_SPARES, needs to be set to 1.

File: Rainier-4U-MRW.xml [hunk 2]
```diff
@@ -93820,6 +93388,10 @@
 		<id>IO_IOHS_PRE2</id>
 		<default>0</default>
 	</attribute>
+	<attribute>
+		<id>IO_IOHS_XTALK</id>
+		<default>NO_XTALK</default>
```
Comment (dcrowell77) [inline Rainier-4U-MRW.xml:93393]: I'm not expecting a new value for this to show up.  Please confirm the value is correct for the link that got changed.

File: Rainier-4U-MRW.xml [hunk 3]
```diff
@@ -94027,6 +93595,10 @@
...
```
Comment (dcrowell77) [inline Rainier-4U-MRW.xml:93600]: Confirm this unexpected new value

**Comment by dcrowell77** [comment]
The SPD changes look right but it looks like something else got pulled in as well on accident.

File: Rainier-4U-MRW.xml (hunk 1 above)
Comment (dcrowell77) [reply Rainier-4U-MRW.xml:50922]: Ignore this comment, I thought I was looking at an Everest update.  A blank value is okay for Rainier (though an explicit zero would still be better.

**Comment by bofferdn** [comment]
ok to test

**Review by dcrowell77 — APPROVED** [review]

**Comment by dcrowell77** [comment]
retest this please

**Comment by causten** [comment]
That's an odd title for a PR.  You sure you don't what to have a better description?   People from other companies have no idea what it is.  Enjoy your day!

**Comment by bofferdn** [comment]
We'll ask the system owners to enumerate the changes better in future commits

**Pull request merged**